tests/PHPUnit/Unit/ConfigTest.php

```diff
@@ -347,19 +268,23 @@ public function getDumpConfigData()
                 array('General' => array('debug' => 0),       // global
                       'Tracker' => array('anonymize' => 0)),
                 array(),                                      // common
-                array('Tracker' => array('anonymize' => 2)),
+                array('Tracker' => array('anonymize' => 2),
+                      'General' => array('debug' => 1)),
                 $header . "[General]\ndebug = 1\n\n[Tracker]\nanonymize = 2\n\n",
             )),
 
-            array('sort, common sections before new section', array(
+            /*TODO: is this test needed?
```

Comment:
It's important that sections are stored in config.ini.php in the same order as they are in global.ini.php 
this is because when we write the dump to file it should put [database] at top, then general, tracker, debug, and after only the plugins etc. we could also hardcode the order of sections if that's easier